packages/core/src/odata/common/request-builder/batch/batch-request-options.ts

```diff
@@ -0,0 +1,13 @@
+import { Destination } from '../../../../scp-cf';
+
+export interface BatchRequestOptions {
```

Comment:
Discussion: The subrequest service path is required. If there will be additional options in the future it could be better to make it optional so that you do not have to give it. Especially with the default in mind.

Reply:
Good catch.